Makefile

```diff
@@ -236,9 +240,10 @@ endif
 ifeq ($(llvm_version),3.7.1)
 else ifeq ($(llvm_version),3.8.1)
 else ifeq ($(llvm_version),3.9.1)
+else ifeq ($(llvm_version),4.0.0)
 else
   $(warning WARNING: Unsupported LLVM version: $(llvm_version))
-  $(warning Please use LLVM 3.7.1, 3.8.1, or 3.9.1)
+  $(warning Please use LLVM 3.7.1, 3.8.1, 3.9.1, or 4.0.0)
```

Comment:
Ok, I wasn't aware of the homebrew status.

IIRC, we're not supporting 3.7.0, 3.8.0 and 3.9.0 because we've found blocking LLVM bugs in these versions. Until we find such a bug in either 4.0.0 or 4.0.1, I think it would be good to officially support both.